clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -3776,6 +3780,9 @@ void CIRGenModule::Release() {
   if (getTriple().isPPC() && !MissingFeatures::mustTailCallUndefinedGlobals()) {
     llvm_unreachable("NYI");
   }
+
+  // Finalize module attributes (including inline assembly)
+  finalizeModuleAttributes();
```

Comment:
OG doesn't do this right here, why are we? There's already other model level attributes being handled, the content of this function should be there